catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -31,7 +31,16 @@
                 name="Maximum Number Pending Results" id="maxPendingResults" required="true" type="Integer"
                 default="10000"
         />
-
+        <AD
+                description="Should the source library portion of the query be removed before execution."
```

Comment:
Add a bit more here - like "....This is necessary since source library maps to sourceId and sourceId is not an attribute of the metacard and will never match on any queries."